cluster/ci/config/prow/config.yaml

```diff
@@ -1098,9 +1098,9 @@ presubmits:
     labels:
       master: ci.openshift.redhat.com
     always_run: false
-    context: ci/openshift-jenkins/unit
-    rerun_command: "/test unit"
-    trigger: "((?m)^/test( all| unit),?(\\s+|$))"
+    context: ci/openshift-jenkins
```

Comment:
Normally, in such a change the old context needs to be migrated to the new context. 

https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus

Reply:
Yeah, unless there is a really compelling reason to do this let's not change the name.